.circleci/config.yml

```diff
@@ -10,19 +10,77 @@ common-steps:
       name: Download wheels and sources
       command: make fetch-wheels
 
-  - &makesourcetarball
+  - &getlatestreleasedversion
+    run:
+      name: Get latest released version of the project
+      command: |
+        cd ~/packaging/securedrop-*
+        export VERSION_TO_BUILD="$(git describe --tags $(git rev-list --tags --max-count=1))"
+        # Enable access to this env var in subsequent run steps
+        echo $VERSION_TO_BUILD > ~/packaging/sd_version
+        echo 'export VERSION_TO_BUILD=$(cat ~/packaging/sd_version)' >> $BASH_ENV
+
+  - &getnightlyversion
     run:
-      name: Get latest tag for the project and make a source tarball
+      name: Create nightly version
       command: |
         cd ~/packaging/securedrop-*
```

Comment:
This line could be problematic if we will use in the same container for multiple builds, but since we are doing 1 container per deb, this will work as expected